Sora/Configuration.swift

```diff
@@ -220,6 +220,10 @@ public struct Configuration {
   /// 通常、指定する必要はありません。
   public var publisherAudioTrackId: String = defaultPublisherAudioTrackId
 
+  /// Sora との接続に利用する CA 証明書のデータです。
```

Comment:
細かいですがデータはなくてもいいかもしれません。
今のままでも通じると思うので修正は必須ではありません。
```suggestion
  /// Sora との接続に利用する CA 証明書です。
```